Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -267,10 +267,14 @@ extension BrowserViewController: WKNavigationDelegate {
             return
         }
 
-        // Ignore JS navigated links, the intention is to match Safari and native WKWebView behaviour.
-        if navigationAction.navigationType == .linkActivated {
```

Comment:
opening a zoom scheme link triggers `case other = -1` navigation type for some reason..
now we match what Firefox does